cmd/status.go

```diff
@@ -165,20 +165,19 @@ func waitForAlligators(ctx context.Context, kubeClient *k8s.Client, helmRelease
 	defer spinner.Stop()
 
 	runningAlligators := 0
-	err := spinner.Poll(
-		func() (bool, error) {
-			var err error
-			runningAlligators, err = getRunningAlligators(ctx, kubeClient, helmRelease.Chart.AppVersion(), helmRelease.Namespace)
-			if err != nil {
-				return false, err
-			}
 
-			spinner.Message(fmt.Sprintf(WAIT_FOR_ALLIGATORS_FORMAT, runningAlligators, expectedAlligatorsCount))
-			return runningAlligators == expectedAlligatorsCount, nil
-		},
-		PODS_POLLING_INTERVAL,
-		ALLIGATORS_POLLING_TIMEOUT,
-	)
+	isAlligatorRunningFunc := func() (bool, error) {
```

Comment:
just to make sure, other then separating this function to a separate declaration, nothing changed other by adding context right?

Reply:
right